### ISSUE TITLE
Nix integration is obnoxious on NixOS

### ISSUE THREAD

**Issue by jeanlucthumm** [oh-my-fish/theme-agnoster#74]
On NixOS you always get the virtual env nix integration even in nornal command line:

![image](https://github.com/user-attachments/assets/e02f3644-5b59-42a9-a428-0c83128e8437)

I couldn't find a way to disable this, would be nice if there was a global option.

**Comment by bakhtiyarneyman** [comment]
Hmm, in "normal command line" meaning you didn't explicitly run `nix shell`/`nix develop`/`nix-shell` prior? I suspect one of those commands is being invoked tacitly by your shell configuration. If we understand what puts packages into yout `$PATH` maybe we can make it work for setups like yours only on explicit invocactions without disabling it altogether.

**Comment by jeanlucthumm** [comment]
Yes that's correct. I'm assuming that running a shell in NixOS triggers a false positive since everything is nix.
